docs/writing-schemas/scalars.md

```diff
@@ -25,39 +25,38 @@ extended scalar types provided by `graphql-java`.
 ## ID
 
 GraphQL supports a the scalar type ID, a unique identifier that is not intended to be human readable. ID's are
```

Comment:
"IDs". No apostrophe.
For some reason, GitHub won't let me edit the file.

Reply:
@DelfCo Did you change your username?

I have added you as a `Triage` maintainer for the repo so you should be able to make suggestions

Reply:
Oh. I see what happened. I was commenting on this one from my desktop at home, which is signed into my personal GitHub account rather than my Expedia Employee GitHub account. Thanks for the add. I'll try to maintain better sign-in discipline in future to avoid confusion about who I am and what I'm doing here.